README.md

```diff
@@ -1164,7 +1166,8 @@ Table of Contents
   * [education.github.com](https://education.github.com/pack) — Collection of free services for students. Registration required
   * [Framacloud](https://degooglisons-internet.org/en/list/) — A list of Free/Libre Open Source Software and SaaS by the French non-profit [Framasoft](https://framasoft.org/en/).
   * [getawesomeness](https://getawesomeness.herokuapp.com) — Retrieve all amazing awesomeness from GitHub... a must see
-  * [github.com — FOSS for Dev](https://github.com/tvvocold/FOSS-for-Dev) — A hub of free and Open Source software for developers
+  * [github.com — FOSS for Dev](https://github.com/tvvocold/FOSS-for-Dev) — A hub of free and Open Source software for developers.
```

Comment:
Please add this to the existing github entry if not already there.

Reply:
@ripienaar I did not add that entry. It already exists. I just added some punctuation to it.

Reply:
If you want I can revert that back.

Reply:
ah indeed, no its fine I missread, no problem

Reply:
No problem. Happens.